doc/live_iso.md

```diff
@@ -49,3 +70,158 @@ Notes:
+
+### Injecting the Default Password Into the ISO Image
+
+Another option is to inject your custom hashed password directly into the ISO
+image. The advantage is than you can easily use the same image for installing
+multiple machines and you do not need to configure anything during the boot.
+
+To inject a new password into the ISO run:
+
+```sh
+# replace the agama.iso name with your image name
+tagmedia --add-tag "agama_password=$((openssl passwd -6) | base64 -w 0)" agama.iso
+```
+
+It will interactively ask for a password then it will be hashed using the SHA512
+algorithm, encoded to the Base64 encoding and appended to the application area
+in the ISO file. If you want to update the password then just the same command
+again, it will overwrite the existing password.
+
+See the [Creating a Hashed Password](#creating-a-hashed-password) section below
+if you want to use a different hashing algorithm than SHA512.
+
+To check all tags present in an ISO file use this command:
+
+```sh
+# replace the agama.iso name with your image name
+tagmedia agama.iso
+```
+
+If you want to remove the password setting from the ISO image then run:
+
+```sh
+# replace the agama.iso name with your image name
+tagmedia --remove-tag agama_password agama.iso
+```
+
+:information_source: *Note: The image usually already contains some other tags,
+like the checksums for verifying the medium integrity. Do not touch them!*
+
+### Random Password as a Fallback
+
+When no password is specified or the entering the password interactively was
+canceled by the user then Agama generates a random password and prints it on the
+console.
+
+### Password Priority
+
+The password setting priority is following (from highest priority to the
+lowest):
+
+1. Password entered interactively during the boot process
+2. Password entered on the boot command line
+3. Default password from the ISO image meta data
+4. A random password is generated as a fallback
+
+### Creating a Hashed Password
+
+There are several ways how to create a password hash, here we will mention two
+tools.
+
+Each tool allows to select the encryption method to use. To check the details
+about all encryption methods see `man 5 crypt`, it lists the encryption methods
+sorted by their strength so you can check which methods are recommended and
+which ones should be avoided.
+
+#### Mkpasswd
+
+You can use the `mkpasswd` tool from the `whois` package. It offers a lot of
+encryption methods, see the `mkpasswd -m help` for the list.
+
+By default it uses the strongest method available so in most cases you just run
+
+```sh
+mkpasswd
+```
+
+and then enter the password on the command line prompt.
+
+#### OpenSSL
+
+Alternatively you can use the `openssl passwd` command from the openSSL package.
+It offers less encryption methods but on the other hand it should be basically
+installed in every system.
+
+:warning: *Warning: By default it uses a weak encryption method (DES or MD5
```

Comment:
GitHub implements a nicer warning message (we are using them already):

```
> [!WARNING]
> Put your message here...
```

Reply:
Nice, there are supported even some more info boxes.
https://github.com/orgs/community/discussions/16925